sycl/test-e2e/Graph/RecordReplay/event_profiling_info.cpp

```diff
@@ -0,0 +1,102 @@
+// REQUIRES: level_zero, gpu
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out 2>&1
+// RUN: %if ext_oneapi_level_zero %{env ZE_DEBUG=4 %{run} %t.out 2>&1 | FileCheck %s %}
+//
+// CHECK-NOT: LEAK
+
+// This test checks the profiling of an event returned
+// from graph submission with event::get_profiling_info().
+// The second run is to check that there are no leaks reported with the embedded
+// ZE_DEBUG=4 testing capability.
+
+#include "../graph_common.hpp"
+#include <cassert>
+#include <iostream>
+#include <sycl/sycl.hpp>
```

Comment:
don't think we need these headers, pretty much all the other tests just only include `#include "../graph_common.hpp"`